tests/rust/roundtrips/image/src/main.rs

```diff
@@ -1,5 +1,8 @@
 //! Logs an `Image` archetype for roundtrip checks.
 
+// Allow unwrap() in tests (allow-unwrap-in-tests doesn't apply)
```

Comment:
that is again because of the ticket you mentioned on the other tests? Would be best to have both the ticket and the synopsis (allow-unwrap-in-tests doesn't work on integration tests) consistently at all of them

Reply:
No, I think this is different - isn't `roundtrip` technically not a test at all but just an app we built to use for testing?

Reply:
ah you're right, I missed that!